desc/integrals/_bounce_utils.py

```diff
@@ -538,6 +558,108 @@ def _plot_check_interp(Q, V, name=""):
         plt.show()
 
 
+def plot_ppoly(
```

Comment:
just realizing, we don't have an example that uses this in the tests? or do we?

Reply:
used in `bounce.plot`